app/scripts/app.js

```diff
@@ -192,11 +192,15 @@ angular.module('confRegistrationWebApp', ['ngRoute', 'ngCookies', 'ui.bootstrap'
                  * server side so the users access_token can be fetched to build the log out URL.
                  * after the GET, if successful, then delete the cookies. */
               } else if ($cookies.crsAuthProviderType === 'FACEBOOK') {
-                $http.get('auth/facebook/logout').success(function (data, status, headers) {
+                $http.get('auth/facebook/logout').success(function () {
```

Comment:
Looks good.  Do the endpoints need to different for different auth providers? Could it just be one /auth/expire or something simliar?